dashboard/app/helpers/allowed_hostname_helper.rb

```diff
@@ -4,107 +4,137 @@
 # - These APIs are allowlisted to let students make web requests in their projects
 # - All APIs should be public, well-documented, and not require authentication for basic usage
 # - APIs that require API keys should be documented as such.  These risk student exposure to the API keys.
```

Comment:
Nit: Could update line 6 to something like 'APIs that require API keys are documented as such.'